### PR TITLE
more developer friendly

### PR DESCRIPTION
**Pull request by jipiboily**
@epaulet

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@ def invalid_request(message = 'invalid request', type: 'invalid_request')
   end
 
   def verify_signature
+    return true if Rails.env.development?
```
Comment (epaulet) [inline app/controllers/events_controller.rb]: Wouldn't this bypass the ability to test whether this method will actually do the needful once we ship it?

Comment (jipiboily) [inline app/controllers/events_controller.rb]: This is why I added https://github.com/rainforestapp/rainforest-integrations/pull/21/files#diff-69b2650604ff734e515c98c3fba94314R161

Maybe it's not complete enough? There are already tests making sure it's doing its job in the `events_controller_spec`. Let me know if that's enough or not.

Comment (epaulet) [inline app/controllers/events_controller.rb]: I meant whether this method will ever be tested for when the environment is not develop, but I see the unit tests now :)

**Comment by epaulet** [comment]
:shipit:

**Pull request merged**